File: library/src/main/java/com/dhy/debugutil/data/ConfigResponse.java
package com.dhy.debugutil.data;

import android.support.annotation.Nullable;

import java.io.Serializable;
import java.util.List;

public class ConfigResponse<CONTENT extends Serializable> implements Serializable {
    private List<ConfigRecord<CONTENT>> results;

    private static class ConfigRecord<CONFIG> implements Serializable {
        List<CONFIG> json;
        String updatedAt;
    }

    @Nullable
    public List<CONTENT> getConfigs() {
        if (results != null && results.size() == 1) {
            return results.get(0).json;
        }
        return null;
    }
}

File: library/src/main/java/com/dhy/debugutil/data/ConfigResponse.kt
package com.dhy.debugutil.data

import java.io.Serializable

class ConfigResponse : Serializable {
    private val results: List<ConfigRecord>? = null

    val configs: List<TestConfig>?
        get() = if (results?.size == 1) results.first().json else null

    private class ConfigRecord : Serializable {
        internal var json: List<TestConfig>? = null
    }
}